### PR TITLE
Documentation about CPython. Make example programs work out of the box.

### PR DESCRIPTION
**Pull request by amotl**
## About
By applying minor adjustments, the example programs are now in a form that supports integration testing with CrateDB.

## Rationale
Based on a project layout like this, it will be easy to add integration tests that validate the package, also considering its "examples" collection as the first and always best set of general test cases. When run to completion on behalf of a test suite, it also yields piece of mind because you always know your example programs stay healthy.

## What's Next
Integration tests and a CI/GHA configuration will be added by a subsequent patch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,25 +46,23 @@ import cratedb
...
-Connect to a CrateDB cluster in the cloud by providing hostname, user name and password:
+Connect to a CrateDB Cloud cluster using SSL, by providing hostname, username, and password:
 
 ```python
 crate = cratedb.CrateDB(
     host="host", 
     user="user", 
-    password="password"
+    password="password",
+    use_ssl=True
```
Comment (simonprickett) [inline README.md:56]: We don't need `use_ssl` by default... by design, as microcontrollers will mostly be connecting to the cloud which requires it so it's on by default to help minimize code size.  Could be a micro-optimization but that was why it's on by default and originally only mentioned in the local network / Docker Crate example.

File: README.md [hunk 2]
```diff
@@ -74,7 +72,8 @@ crate = cratedb.CrateDB(
     host="host", 
     user="user", 
     port=4201,
-    password="password"
+    password="password",
+    use_ssl=True
```
Comment (simonprickett) [inline README.md:76]: Can drop `use_ssl` as it's on by default.

File: cratedb.py [hunk 3]
```diff
@@ -85,7 +85,7 @@ class CrateDBError(Exception):
     pass
 
 class CrateDB:
-    def __init__(self, host, port=4200, user=None, password=None, schema="doc", use_ssl=True):
+    def __init__(self, host, port=4200, user=None, password=None, schema="doc", use_ssl=False):
```
Comment (simonprickett) [inline cratedb.py:88]: There was a good reason why it was True, I disagree with this.

File: docs/cpython.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
+# Running on CPython
+
+The module is being conceived for [MicroPython], but also works on [CPython].
```
Comment (simonprickett) [inline docs/cpython.md:3]:
```suggestion
The module is designed for [MicroPython], but also works on [CPython].
```

File: examples/example_usage.py [hunk 5]
```diff
@@ -6,14 +6,17 @@
 import cratedb
 
 # CrateDB Docker / local network, no SSL.
-# crate = cratedb.CrateDB(host="hostname", use_ssl=False)
+crate = cratedb.CrateDB(host="localhost")
```
Comment (simonprickett) [inline examples/example_usage.py:9]: This should use `use_ssl=False`.

File: examples/example_usage.py [hunk 6]
```diff
@@ -6,14 +6,17 @@
...
+# CrateDB Cloud, using SSL.
+"""
 crate = cratedb.CrateDB(
-    host="host", 
-    user="user", 
-    password="password"
+    host="testdrive.cratedb.net",
+    user="username",
+    password="password",
+    use_ssl=True
```
Comment (simonprickett) [inline examples/example_usage.py:17]: Remove, as `True` should be the default.

File: examples/object_examples.py [hunk 7]
```diff
@@ -6,14 +6,17 @@
 import cratedb
 
 # CrateDB Docker / local network, no SSL.
-# crate = cratedb.CrateDB(host="hostname", use_ssl=False)
+crate = cratedb.CrateDB(host="localhost")
```
Comment (simonprickett) [inline examples/object_examples.py:9]: Add `use_ssl=False`

File: examples/object_examples.py [hunk 8]
```diff
@@ -6,14 +6,17 @@
...
+# CrateDB Cloud, using SSL.
+"""
 crate = cratedb.CrateDB(
-    host="host", 
-    user="user", 
-    password="password"
+    host="testdrive.cratedb.net",
+    user="username",
+    password="password",
+    use_ssl=True
```
Comment (simonprickett) [inline examples/object_examples.py:17]: Omit, as True should be the default.

**Review by simonprickett — CHANGES_REQUESTED** [review]
Looks good, but don't change the default for `use_ssl`.  Having it default to true covers the most common use case and minimizes extra code on microcontrollers.

File: docs/cpython.md [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+```
+
+## Usage
+
+Start CrateDB.
+```shell
+docker run --rm -it --name=cratedb \
+  --publish=4200:4200 --publish=5432:5432 \
+  --env=CRATE_HEAP_SIZE=2g crate:latest -Cdiscovery.type=single-node
+```
```
Comment (amotl) [inline docs/cpython.md:25]: Should also add this snippet to the main README?

**Comment by amotl** [comment]
> Looks good, but don't change the default for `use_ssl` [in the library].

Thanks. I've just reverted corresponding changes, still staging the example files ready to be invoked on a non-SSL instance.

File: docs/cpython.md (hunk 9 above)
Comment (amotl) [reply docs/cpython.md:25]: Done. ✅

**Pull request merged**